src/store/store.ts

```diff
@@ -0,0 +1,25 @@
+import { combineReducers, configureStore } from "@reduxjs/toolkit";
+import { api } from "../services/ApiService";
+import currentPageItemsReducer from "./slices/currentPageSlice";
+import selectedBooksReducer from "./slices/selectedBooksSlice";
+import selectedItemDetailsReducer from "./slices/selectedItemDetailsSlice";
```

Comment:
```suggestion
import currentPageItems from "./slices/currentPageSlice";
import selectedBooks from "./slices/selectedBooksSlice";
import selectedItemDetails from "./slices/selectedItemDetailsSlice";
```
ты ж их сразу можешь назвать как тебе надо